### PR TITLE
added Persian Locale

### PR DESCRIPTION
**Pull request by ali-sokhandan**

### CODE REVIEW

File: locale/fa_IR/countries.xml [hunk 1]
```diff
@@ -0,0 +1,269 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE countries SYSTEM "../../dtd/countries.dtd">
+
+<!--
+  * lib/pkp/locale/fa_IR/countries.xml
+  *
+  * Copyright (c) 2014-2016 Simon Fraser University Library
```
Comment (asmecher) [inline locale/fa_IR/countries.xml]: Copyrights should be to Simon Fraser University, not Simon Fraser University Library.

Comment (asmecher) [reply locale/fa_IR/countries.xml]: All copyright dates for Simon Fraser University and John Willinsky should end -2018.

File: locale/fa_IR/common.xml [hunk 2]
```diff
@@ -0,0 +1,531 @@
...
+    <message key="notification.removedSubmission">مقاله ارسالی حذف شد.</message>
+    <message key="notification.notification">اطلاع‌رسانی</message>
+    <message key="notification.notifications">اطلاع‌رسانی‌ها</message>
+    <message key="notification.notificationsNew">({$numNew} جدید)</message>
+    <message key="notification.notificationsDescription">شما {$unreadCount} مورد اطلاعیه ناخوانده دارید و {$readCount} مورد خوانده شده.  <![CDATA[  <a href="{$settingsUrl}">Click here</a> to change your notification settings.]]></message>
+    <message key="notification.sentNotification">اطلاع‌رسانی ارسال شد</message>
+    <message key="notification.settings">تنظیمات اطلاع‌رسانی</message>
+    <message key="notification.settingsDescription">لطفاً رویدادی از سیستم را که مایلید در مورد آن به شما اطلاع‌رسانی شود انتخاب نمایید. عدم انتخاب یک رویداد سبب می‌گردد که اطلاع‌رسانی آن در فیلد اطلاع‌رسانی شما ظاهر نشود.</message>
+    <message key="notification.subscribe">مشترک شدن</message>
+    <message key="notification.subscribeError">ایمیل شما قبلاً برای ارسال اطلاع‌رسانی ثبت شده بود.</message>
```
Comment (asmecher) [inline locale/fa_IR/common.xml]: If this is a pain to change, don't worry about it -- but we prefer tab-based indents, not space-based indents.

File: locale/fa_IR/default.xml [hunk 3]
```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../dtd/locale.dtd">
+<!--
+  * locale/en_US/default.xml
```
Comment (asmecher) [inline locale/fa_IR/default.xml]: Double-check the path names to make sure they're correct -- this one should be `locale/fa_IR/default.xml`, not `locale/en_US/default.xml`.

File: plugins/generic/acron/locale/fa_IR/locale.xml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../../dtd/locale.dtd">
+<!--
+  * locale.xml
+  *
+  * Copyright (c) 2013-2017 Simon Fraser University
+  * Copyright (c) 2000-2017 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  *
+  * Localization strings for the en_US (U.S. English) locale.
```
Comment (asmecher) [inline plugins/generic/acron/locale/fa_IR/locale.xml]: Update the text here for the right locale name (or better yet, remove it -- I've just done that from [the English text](https://github.com/pkp/pkp-lib/commit/21623a3406ecdbc2559913a58ef8243514bd0864), as it's just extra maintenance without adding value.)

**Comment by ali-sokhandan** [comment]
dear @asmecher , the mentioned changes has been applied.

**Comment by asmecher** [comment]
(Noted at https://github.com/pkp/pkp-lib/issues/3320)

**Comment by bozana** [comment]
It seems that a few files contain `"` in the text, instead of maybe `&quot;` : 

./lib/pkp/locale/fa_IR/bic21subjects.xml:184: parser error : attributes construct error
	<subject code="AVGC" text="موسیقی غرب "کلاسیک"" />
	                                                ^
./lib/pkp/locale/fa_IR/bic21subjects.xml:184: parser error : Couldn't find end of Start Tag subject line 184
	<subject code="AVGC" text="موسیقی غرب "کلاسیک"" />
	                                                ^
./lib/pkp/locale/fa_IR/bic21subjects.xml:194: parser error : attributes construct error
	<subject code="AVGE" text="موسیقی غیر غربی: سنتی و "کلاس�
	                                                                      ^
./lib/pkp/locale/fa_IR/bic21subjects.xml:194: parser error : Couldn't find end of Start Tag subject line 194
	<subject code="AVGE" text="موسیقی غیر غربی: سنتی و "کلاس�
	                                                                      ^
./lib/pkp/locale/fa_IR/bic21subjects.xml:204: parser error : attributes construct error
	<subject code="AVGQ" text="روح و R "n" B" />
	                                        ^
./lib/pkp/locale/fa_IR/bic21subjects.xml:204: parser error : Couldn't find end of Start Tag subject line 204
	<subject code="AVGQ" text="روح و R "n" B" />

@ali-sokhandan, could you fix it?

**Comment by ali-sokhandan** [comment]
hi
I want to know that is the locale files update by the OJS official member or it can updated by others.
and if the second case, how can I update the locale to proper for new OJS version?
thanks
@asmecher @bozana

**Comment by bozana** [comment]
Hi @ali-sokhandan, I am not sure I understand the question correctly, but: could you make a PR, as you did it here last time? Then we will take a look and merge...

**Comment by ali-sokhandan** [comment]
dear @bozana yes of course, but my problem is that how can I find the new messages or changed ones (which should translated) on the coming version (3.1.1) or i should wait for its release.

**Pull request merged**